src/modules/dashboard/sagas/task.ts

```diff
@@ -196,10 +196,6 @@ function* taskCommentAdd({
   meta,
 }: Action<ActionTypes.TASK_COMMENT_ADD>) {
   try {
-    yield call(signMessage, 'taskComment', {
-      comment,
-      author,
-    });
```

Comment:
I know there's a chance that we'll bring this feature back at some point, but I think we can remove this saga (for now at least), no? And just handle the mutation directly in the component?

Reply:
Yes! When we bring it back it won't be handled in a saga

Reply:
Good point. I'll make the change